### PR TITLE
Added answer for polymorphism

### PR DESCRIPTION
**Pull request by saira512dev**
I have added an answer to "Polymorphism" on branch "answer". Do check and merge if you find it correct. Thanks

### CODE REVIEW

File: BANKI-answers.md [hunk 1]
```diff
@@ -1451,10 +1451,42 @@ function factorial(num) {
 
 - [ ] What is polymorphism?
 
-  - **Explanation:**
-  - **Use:**
+  - **Explanation:** Polymorphism is a concept of Object Oriented Paradigm(OOP) that provides a way to perform a single action in different ways.
```
Comment (curtisbarnard) [inline BANKI-answers.md]: I think this should be "Object-oriented programming (OOP) paradigm"

**Review by curtisbarnard — CHANGES_REQUESTED** [review]
Thanks for the submission @saira512dev. Just one little tweak on a single line is needed then' I'll get this merged.

File: BANKI-answers.md (hunk 1 above)
Comment (saira512dev) [reply BANKI-answers.md]: I have made the suggested change. Thanks!

**Review by curtisbarnard — APPROVED** [review]
Looks great, thanks!

**Pull request merged**